### PR TITLE
Prepare release 3.1.0-beta.1, migrate to Swift 5

### PR DESCRIPTION
**Pull request by Kaspik**
This PR is here to track all the changes on development before merging it and releasing new version 3.1.0 from master.

Please don't merge this, just optionally review all changes so we can discuss the proper time for release.

Things to do before merging:
- review and update README
- review and update documentation
- verify tests are running

### CODE REVIEW

**Review by alexmomotiuk — APPROVED** [review]

**Comment by apozdeyev** [comment]
Hi All! 
First of all, thank you for the awesome project!
Can you please clarify is it possible to use the latest version of InputBarAccessoryView (4.3.2) in current release? It contains useful [feature](https://github.com/nathantannar4/InputBarAccessoryView/pull/67) for customization accessoryView's frame insets.

**Comment by Kaspik** [comment]
@apozdeyev Hey! The problem currently is that InputBarAccessoryView 4.3.2 is not on cocoapods - the latest version is 4.3.1.

If @nathantannar4 is able to fix this, we will be more than happy to update to it - either before or after releasing 3.1.0. :)

**Review by alexmomotiuk — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,49 @@
 
 The changelog for `MessageKit`. Also see the [releases](https://github.com/MessageKit/MessageKit/releases) on GitHub.
 
+## Upcoming release
```
Comment (austinwright) [inline CHANGELOG.md]: Can we omit lines 5-15 from this PR? We can have an `## Upcoming release` section on the /development branch, but I don't think it makes sense to have it on /master. It is the first thing a framework users sees, and as-is it looks like a forgotten placeholder.

**Review by Kaspik — COMMENTED** [review]
PR itself looks good to me, approved for merging and releasing beta.

**Review by alexmomotiuk — APPROVED** [review]

**Review by austinwright — APPROVED** [review]

**Pull request merged**